paasta_tools/marathon_tools.py

```diff
@@ -993,3 +1015,45 @@ def kill_task(client, app_id, task_id, scale):
             return []
         else:
             raise
+
+
+def get_instances_from_zookeeper(service, instance):
+    with ZookeeperPool() as zookeeper_client:
+        (instances, _) = zookeeper_client.get('/autoscaling/%s/%s/instances' % (service, instance))
+        if instances is None or instances == 'None':
+            raise NoNodeError
```

Comment:
it seems odd to do this - if kazoo has decided to not raise exceptions at this point, why're we modifying that? how about we return None here, and change your try/catch to check for None?